### PR TITLE
declaring the tempid default tag

### PR DESCRIPTION
**Pull request by souenzzo**

### CODE REVIEW

File: src/main/fulcro/tempid.cljc [hunk 1]
```diff
@@ -1,6 +1,8 @@
 (ns fulcro.tempid
   #?(:clj (:import [java.io Writer])))
 
+(def tag "fulcro/tempid")
```
Comment (awkay) [inline src/main/fulcro/tempid.cljc]: If you want to add this, I'd prefer it be authoritative...that is, used within the source instead of just declared.

Comment (souenzzo) [reply src/main/fulcro/tempid.cljc]: done! :)

**Comment by souenzzo** [comment]
Now I'm using the tag from tempid in transit ns's too.

**Comment by awkay** [comment]
Cool, thanks

**Pull request merged**